en/docs/apis/api-concepts.md

```diff
@@ -1,23 +1,69 @@
 
-## API Management Components
-Described below are the key components of Choreo API management:
+## API format
+OpenAPI Specification (OAS, a.k.a Swagger) format is the underlying representation of an API in Choreo. You can import an API using a Swagger definition in the Choreo console. Additionally, a user can view, edit, import, or download an API definition in OAS format via the Choreo console.
 
-### API Publisher 
-The API Publisher is designed for API creators to develop, document, secure, test, and version APIs with ease.
+## API resource path and HTTP methods
+An API comprises one or more resources, each of which has a unique resource path (URI). An API Resource has a set of HTTP methods that operates on it. The supported HTTP methods are `GET`, `POST`, `PUT`, `DELETE`, `PATCH`, `HEAD`, and `OPTIONS`.
 
-### Developer Portal
-The Developer portal allows API publishers to host and advertise their APIs while allowing API consumers to self-register, discover, evaluate, subscribe to, and consume APIs securely and easily.
+## API lifecycle
+An API lifecycle comprises the stages an API goes through from creation to retirement. API lifecycle is independent of its backend service. The lifecycle states are: `Created`, `Prototyped`, `Published`, `Deprecated`, `Blocked`, `Deployed`, and `Retired`.
 
-### API Key Manager
-API Key Manager handles all security and key-related operations. 
+## Application
+An Application in Choreo is the logical representation of a physical application such as a mobile app, web app, device, etc. For an application to use an API, the application should subscribe to the relevant APIs it intends to use. A subscription to an API happens over a selected business plan.  The business plan determines the usage quota the subscription gets. An application has a consumer key and a consumer secret which acts as the credentials of the application.
 
-### API Gateway
-API Gateway acts as the entry point for an API request. It secures, protects, manages, and scales API calls. 
+## Access token
+Choreo uses the OAuth2.0 bearer token-based authentication mechanism to allow a consumer to access an API.  
+The access token is a string that the consumer passes in an HTTP header of the API request. Choreo supports JWT formatted self-contained access tokens.
 
-## API Lifecycle
-API lifecycle comprises of the stages an API goes through from creation to retirement. API lifecycle is independent of its backend service. The lifecycle states are `Created`, `Prototyped`, `Published`, `Deprecated`, `Blocked`, and `Retired`.
+## Rate limiting
+Rate limiting makes it possible to limit the number of permitted requests to an API for a given time window.
+Thereby it allows you to:
```

Comment:
`When a colon introduces a list, the text that precedes the colon must be able to stand alone as a complete sentence.`
Seems we need to follow the above for this sentence

Reply:
reworded.